### DISCUSSION TITLE
Object Hydration Failing on Enum with Default Value.

### DISCUSSION THREAD

**Discussion by michaeldnelson** [doctrine/doctrine2#9437] [Support Questions] [answered]
```php
#[ORM\Column(type: 'string', length: 20, enumType: Visibility::class)]
public Visibility $visibility = Visibility::PRIVATE;
```
results in:

```
Proxies\__CG__\Model\Entities\Visibility not found
```

Does anyone know if this case should function? Must I include an options default value? I prefer to manage my defaults in code.

I have tried rebuilding my proxy cache.

**Comment by beberlei** [comment, marked as answer]
This is already discussed in this tickets https://github.com/doctrine/orm/issues/9375 and https://github.com/doctrine/orm/issues/9371

**Reply by michaeldnelson** [reply]
Thanks! I actually commented on #9375 and was told it had nothing to do with that ticket.  It does indeed look like #9371 though, thanks for your help! I can work around it for the time being.
